addon/templates/components/nypr-o-gallery-overlay/slide.hbs

```diff
@@ -60,6 +60,12 @@
               @url={{this.shareURL}}
               @ariaLabel="Share on Facebook"
               @utm={{@share.facebook.utm}}
+            />
+              <tools.share
```

Comment:
FYI this line is throwing a test error because of it's weird indenting - remove two spaces so `<tool.share` lines up with the line above it and that should resolve it.